sys/mips/mips/elf_machdep.c

```diff
@@ -159,11 +159,12 @@ static __ElfN(Brandinfo) freebsd_brand_info = {
 	.emul_path	= NULL,
 	.interp_path	= "/libexec/ld-elf.so.1",
 	.sysvec		= &elf_freebsd_sysvec,
-	.interp_newpath	= NULL,
 #if __has_feature(capabilities)
+	.interp_newpath	= "/libexec/ld-cheri-elf.so.1",
 	.header_supported = mips_elf_header_supported,
 	.flags		= BI_CAN_EXEC_DYN
 #else
+	.interp_newpath	= NULL,
 	.brand_note	= &__elfN(freebsd_brandnote),
```

Comment:
Curious that this is ELF64-only; for RISC-V, the brand note stuff has been left as the same for both and things seem to work...

Reply:
I think I had just preserved this from one in sys/mips/cheri/cheriabi_machdep.c.  It probably should work for MIPS as well.